app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -22,6 +22,15 @@ class Api::V2::ContentViewsController < Api::V2::ApiController
       param :solve_dependencies, :bool, :desc => N_("Solve RPM dependencies by default on Content View publish, defaults to false")
     end
 
+    def filtered_associations
```

Comment:
Did you add the filtered_associations here to serve as kind of a documentation? I was going to ask to move it to the private block down below but I noticed it was right next to the content_view param group

Reply:
Yeah, i had put it at the top so it would be somewhat obvious, and next tot he apidoc declaration you can easily match up the associations from the apidoc.  I could see it here or down where the 'view_params' method is defined.  IT couldn't be private because its being called from its parents class.  It could be protected though

Reply:
Cool. I'm good with that